src/other/utils.jl

```diff
@@ -49,7 +49,8 @@ function makeidentifier(s::AbstractString)
     return String(take!(res))
 end
 
-function make_unique(names::Vector{Symbol}; allow_duplicates=true)
+# TODO: change makeunique to false after deprecation period
+function _makeunique(names::Vector{Symbol}; makeunique::Bool=true)
```

Comment:
Why rename the function?

Reply:
I renamed it when we had `make_unique` keyword. Now it is `makeunique` so there would be no name clash.

I understood that `_` signals that the function is for internal use only. So based on my understanding `_makeunique` should be an OK name. I can change it if there are some guidelines for this.

Reply:
I'd better keep it as is, as there's no strong reason to change it and I don't think we systematically use the `_` prefix for internal variables.

Reply:
fixed